client/src/services/deleteReview.js

```diff
@@ -1,11 +1,11 @@
-const deleteReview = async (userId, reviewId, reviewUserID) => {
+const deleteReview = async (reviewId, reviewUserID) => {
   try {
     const deletedRows = await fetch(`/api/v1/reviews/${reviewId}`, {
       method: "DELETE",
       headers: new Headers({
         "Content-Type": "application/json",
       }),
-      body: JSON.stringify({ userId, reviewId, reviewUserID }),
+      body: JSON.stringify({ reviewId, reviewUserID }),
```

Comment:
there is no data that needs to be sent in the body - the `reviewId` in the api request will tell us which review we want to delete - `/api/v1/reviews/${reviewId}`